README.md

```diff
@@ -76,111 +74,19 @@ npm run deploy
 
 See the [Nuxt documentation](https://nuxtjs.org/deployments/github-pages/) for more details.
 
-### Developer information
-
-**use node v16.x LTS**!
-
-:warning: 
-The Annotation Tool is built with the Nuxt framework and currently depends on Nuxt2. 
-Nuxt2 does not support node versions beyond the v16 LTS (see e.g. [this Github issue](https://github.com/nuxt/nuxt/issues/10844)).
-If you want to run the tool locally, make sure you are using node v16.x.
-A good way to manage different node versions is to use the [node version manager](https://github.com/nvm-sh/nvm) tool.
-
-## Annotation Workflow
+## Workflow
 
 1. Upload data table (and/or data dictionary)
-2. Linking columns you want annotated with Neurobagel metadata categories
+2. Link columns you want annotated with Neurobagel metadata categories
 3. Annotate the values of the those categirzed columns
 4. Download an annotated version of a BIDS-style data dictionary for your dataset
 
-## General Navigation
-
-<img width="300" src="./images/readme_navbar.png" alt="Annotation tool navbar screenshot">
-<img width="300" src="./images/readme_nextpage_button.png" alt="Annotation tool next page button screenshot">
-
 There are two means of moving forward to the next step in the annotation tool's workflow: (1) the <span style="color:green;">navbar</span> at the top right of the screen which features the page names, and (2) the <span style="color:green;">next page buttons</span> on the bottom right of each page.
 
```

Comment:
Should we remove this part as well?

Reply:
I wanted to keep those in since its currently a limitation of the app and its good to have it documented somewhere until its addresses. See https://github.com/neurobagel/annotation_tool/issues/619